clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -723,9 +726,9 @@ DPCT_ENUM_OPTION(
     DPCT_OPTION_VALUES(
         DPCT_OPTION_ENUM_VALUE(
             "free-function-queries", int(ExperimentalFeatures::Exp_FreeQueries),
-            "Experimental extension that allows getting 'id', "
+            "DEPRECATED : Experimental extension that allows getting 'id', "
             "'item', 'nd_item', 'group', and\n"
-            "'sub_group' instances globally.",
+            "'sub_group' instances globally. The free-function-queries will be enabled as default.",
```

Comment:
```suggestion
            "'sub_group' instances globally. ",
```

Reply:
Done, Add "Deprecate this option as the feature is **turned** on by default.","